src/clj/rems/poller/email.clj

```diff
@@ -31,9 +32,18 @@
   (str (:public-url env) "accept-invitation?token=" token))
 
 (defn- application-id-for-email [application]
-  (case (util/getx env :application-id-column)
-    :external-id (:application/external-id application)
-    :id (:application/id application)))
+  (str
+   (case (util/getx env :application-id-column)
+     :external-id (:application/external-id application)
+     :id (:application/id application))
```

Comment:
I'd actually like to have a comma between the id and the description -- or a similar separator, but I think comma is the least conspicuous.

Reply:
fine by me